schema.sql

```diff
@@ -1,5 +1,12 @@
 /* Database schema to keep the structure of entire database. */
+CREATE DATABASE vet_clinic;
 
 CREATE TABLE animals (
-    name varchar(100)
+    id INT,
+    name VARCHAR,
+    date_of_birth DATE,
+    escape_attempts INT,
+    neutered BOOL,
+    weight_kg DECIMAL,
+    PRIMARY KEY(id)
 );
```

Comment:
- [ ] As requested by the previous reviewer concerning the spelling error in your `animals` table

![image](https://github.com/clin2on3mun/VetClinic-database/assets/78251927/1260d98e-1254-4e23-8171-cb31f2643a35)

- [ ] According to the project requirements 👀

![image](https://github.com/clin2on3mun/VetClinic-database/assets/78251927/4c9956f4-cedc-47d4-89ec-1f8217defeae)

![image](https://github.com/clin2on3mun/VetClinic-database/assets/78251927/12841850-64ba-41f7-973f-308418b5853f)

- [ ] Kindly copy the SQL you used to rename the column into the schema.sql file 🤓👍

- [ ] To rename a column of a table, we use the [ALTER TABLE](https://www.postgresqltutorial.com/postgresql-tutorial/postgresql-alter-table/) statement with RENAME COLUMN clause as follows:

```SQL
ALTER TABLE table_name 
RENAME COLUMN column_name TO new_column_name;
```